README.md

```diff
@@ -1,13 +1,13 @@
 # MRtrix
 
-[![Build Status](https://travis-ci.org/MRtrix3/mrtrix3.svg?branch=updated_syntax)](https://travis-ci.org/MRtrix3/mrtrix3)
+[![Build Status](https://github.com/jdtournier/mrtrix3/workflows/checks/badge.svg)](https://github.com/jdtournier/mrtrix3/actions)
```

Comment:
May be simplest to edit this manually to point to the *MRtrix3* repo rather than your fork prior to merge.

Reply:
Good point...